src/LinAlg/hiopVectorRajaPar.hpp

```diff
@@ -77,6 +77,7 @@ class hiopVectorRajaPar : public hiopVector
 
   virtual void setToZero();
   virtual void setToConstant( double c );
+  virtual void set_to_random_constant(double minv, double maxv) {assert(false&&"not implemented yet!");}
```

Comment:
we need this implementation rather sooner than later. A quick follow-up PR would be a good idea